notebooks/validation_notebooks/new_and_old_lsst_like_configuration_comparision.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<p>can you make this plot as a function of redshift? I am surprised that the exponential cutoff is so different for the two Schechter functions, instead of only the faint-end slope being different and the normalization</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>